src/contract/mod.rs

```diff
@@ -1,12 +1,17 @@
 //! Ethereum Contract Interface
 
-use crate::api::{Accounts, Eth, Namespace};
+#[cfg(feature = "signing")]
```

Comment:
Could you please group all signing related things into a submodule here as well? Basically I'd like a single `#[cfg(feature = "signing")]` stament instead of sprinkling it all around.